frontend/src/app/components/containers/complaints/complaint-filter.tsx

```diff
@@ -320,6 +327,28 @@ export const ComplaintFilter: FC<Props> = ({ type }) => {
             </div>
           </div>
         )}
+        {UserService.hasRole(Roles.CEEB) && (
```

Comment:
Would this be better as a feature flagged component?  

e.g. from call-details.tsx

![image](https://github.com/user-attachments/assets/93e0c9b2-31de-48b3-a93a-a45c47e1edc1)

Reply:
I don't have a great sense of who needs this filter now vs who will need it in the near future. Does the feature flag approach make more sense here based on what we've heard on that front?